src/client/components/MealReview/MealReviewDisplay.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from "react";
+import './mealreview.css';
+const EachMealReviewDisplay = ({ meal }) => {
```

Comment:
You are not using anything from {meal} apart from the id. 
Pass just the id! 💡

Reply:
Okay 👍🏻 will do ,thanks for pointing out.